packages/components/ContextualMenu/src/ContextualMenuItem.tsx

```diff
@@ -0,0 +1,92 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { Image, View } from 'react-native';
+import { IContextualMenuItemSlotProps, IContextualMenuItemState, IContextualMenuItemProps, IContextualMenuItemRenderData, contextualMenuItemName, IContextualMenuItemType } from './ContextualMenuItem.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { Text } from '@fluentui-react-native/text';
+import { settings } from './ContextualMenuItem.settings';
+import { backgroundColorTokens, borderTokens, textTokens, foregroundColorTokens } from '@fluentui-react-native/tokens';
+import { filterViewProps, filterImageProps } from '@fluentui-react-native/adapters';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { useAsPressable, useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
+
+export const ContextualMenuItem = compose<IContextualMenuItemType>({
+  displayName: contextualMenuItemName,
+  usePrepareProps: (userProps: IContextualMenuItemProps, useStyling: IUseComposeStyling<IContextualMenuItemType>) => {
+    const {
+      icon,
+      text,
+      accessibilityLabel = userProps.text,
```

Comment:
The user should have the option to give an alternate accessibilityLabel other than the text of the item. See Checkbox/RadioButton for example. They have an "ariaLabel" prop, and then set: "accessibilityLabel: ariaLabel ?? content"

Reply:
I would agree. I can't really think of an example where a user wouldn't want the accessibilityLabel to be the same as the text other than if they didn't provide text and just an icon (kind of an odd decision but it could be done, right?). In that case, you'd want to allow the user to describe the icon. I also think it's good to allow users to customize the text if they need to. 

On another note, is there a reason why we used the name "ariaLabel" on Checkbox/RadioButton? Could it not have been "accessibilityLabel"?

Reply:
That's what web calls it and what it's more widely known as, I guess I forgot to change it back though. Next time I work on either of those I'll change it :)

Reply:
Thanks Sammy!

Reply:
The current implementation actually lets users set accessibilityLabel themselves. When they are not set, it just defaults to Text.

Reply:
https://wesbos.com/destructuring-default-values in case anyone is curious about default values when destructuring